### ISSUE TITLE
GPG signed commits/tags

### ISSUE THREAD

**Issue by ggioffreda** [vaklinov/zcash-swing-wallet-ui#15]
Hi,
thanks for the first usable Zcash wallet, keep up the good work!

Can I suggest signing the commits or at least the tags? When it comes to wallets for crypto currencies just the checksum could be not enough. Every guarantee about the authenticity of the source code helps build trust and it's really easy to sign commits and tags. Here are a couple of links:

https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work
https://help.github.com/articles/signing-commits-using-gpg/

In a nutshell it's about doing like these guys are doing for Zcash: 
![screenshot from 2016-12-06 21-15-28](https://cloud.githubusercontent.com/assets/711059/20917933/44d296dc-bbf9-11e6-84b1-7f06cd20f673.png)

Cheers,
Giovanni

**Comment by vaklinov** [comment]
I have started signing the tags that coincide with periodic releases. Signed tags will be named x.xx-SNAPSHOT e.g. :
https://github.com/vaklinov/zcash-swing-wallet-ui/releases/tag/0.46-SNAPSHOT
I will also be tagging/signing milestones even if they are not (binary) releases. I hope this is enough for additional assurance for now. I use eclipse for commits and it seems that it does not yet support signed commits.
